cumulus/parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/teleport.rs

```diff
@@ -357,3 +357,16 @@ fn teleport_native_assets_from_system_para_to_relay_fails() {
 	// Receiver's balance does not change
 	assert_eq!(receiver_balance_after, receiver_balance_before);
 }
+
+// TODO: uncomment when Collectives and BridgeHubWestend are implemented
```

Comment:
link to an issue would be good, if any.

Reply:
This PR adds Westend Collectives. Not sure about BH plans.
https://github.com/paritytech/polkadot-sdk/pull/1737

Reply:
> This PR adds Westend Collectives. Not sure about BH plans. #1737

BridgeHubWestend plans are covered here: https://github.com/paritytech/parity-bridges-common/issues/2602

> PR for BridgeHubWestend does not exists yet

yes, PR does not exist yet, we are waiting for merging https://github.com/paritytech/polkadot-sdk/pull/1215  and then https://github.com/paritytech/parity-bridges-common/issues/2602